server/routerlicious/packages/protocol-definitions/src/consensus.ts

```diff
@@ -36,8 +36,24 @@ export type IApprovedProposal = { approvalSequenceNumber: number } & ISequencedP
  */
 export type ICommittedProposal = { commitSequenceNumber: number } & IApprovedProposal;
 
+/**
+ * A proposal that has been propposed, but not yet accepted or committed
+ */
 export interface IPendingProposal extends ISequencedProposal {
+    /**
+     * Sends a rejection for the proposal
+     */
     reject();
+
+    /**
+     * Disables the sending of rejections for this proposal
+     */
+    disableRejection();
```

Comment:
It doesn't look like the disableRejection is preserved when serialized/deserialized, so other clients will not understand that this proposal should not be rejected.

It seems more like we should special-case proposals that cannot be rejected by adding it to their instantiation i.e. constructor/sendProposal call. Then we need to add some flag on the serialized op so that all clients agree it is accepted regardless of other ops.

We could also have it be accepted on proposal op, rather than on msn >= proposal seq, but I'd lean away from that for now to keep it similar to existing quorum specs.